### PR TITLE
New package: ClipSock.ClipSock version 1.0.1.0

### PR DESCRIPTION
**Pull request by sstallion**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135865)

### CODE REVIEW

**Comment by sstallion** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 639.)

**Comment by stephengillie** [comment]
Manual validation ended with: 
image

File: manifests/c/ClipSock/ClipSock/1.0.1.0/ClipSock.ClipSock.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ClipSock.ClipSock
+PackageVersion: 1.0.1.0
+InstallerLocale: en-US
+InstallerType: wix
+Installers:
```
Comment (stephengillie) [inline manifests/c/ClipSock/ClipSock/1.0.1.0/ClipSock.ClipSock.installer.yaml:8]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

File: manifests/c/ClipSock/ClipSock/1.0.1.0/ClipSock.ClipSock.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+InstallerLocale: en-US
+InstallerType: wix
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/sstallion/ClipSock/releases/download/v1.0.1/ClipSock-1.0.1-x64.msi
+  InstallerSha256: 5170DA6132F07B8952919BAE3985BB3CB6F266820A4EDD166FDBF91DAD1ED0E6
+  ProductCode: '{564B2E18-ED0E-4E4B-8200-D436080D0B68}'
+- Architecture: x86
+  InstallerUrl: https://github.com/sstallion/ClipSock/releases/download/v1.0.1/ClipSock-1.0.1-x86.msi
+  InstallerSha256: B88893AA809656CB16D741497F06CDE248170EFF4A2A972A58ABFD0DE6D32EB6
```
Comment (stephengillie) [inline manifests/c/ClipSock/ClipSock/1.0.1.0/ClipSock.ClipSock.installer.yaml:15]:
```suggestion
  InstallerSha256: 5170DA6132F07B8952919BAE3985BB3CB6F266820A4EDD166FDBF91DAD1ED0E6
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
  ProductCode: '{564B2E18-ED0E-4E4B-8200-D436080D0B68}'
- Architecture: x86
  InstallerUrl: https://github.com/sstallion/ClipSock/releases/download/v1.0.1/ClipSock-1.0.1-x86.msi
  InstallerSha256: B88893AA809656CB16D741497F06CDE248170EFF4A2A972A58ABFD0DE6D32EB6
  Dependencies:
    PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

Comment (sstallion) [reply manifests/c/ClipSock/ClipSock/1.0.1.0/ClipSock.ClipSock.installer.yaml:15]: LGTM. Thank you very much!

**Comment by sstallion** [comment]
I'm not quite sure what's happening with the pipeline, but I'm going to kick off another round of checks to see if this is just a temporary issue.

**Comment by sstallion** [comment]
Looking at the manifest more closely, it looks like I may have fat fingered the version number. For the sake of simplicity, I will close this PR and submit a new package shortly. Sorry for the confusion!

**Pull request closed without merging**